symfony/src/Service/RdrApiService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Service;
+
+use App\Service;
+use Pmi\Entities\Configuration as DatastoreConfiguration;
+use Google_Client as GoogleClient;
+use Google_Service_Oauth2 as GoogleServiceOauth2;
+use Pmi\HttpClient;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class RdrApiService
```

Comment:
In most cases, this would be injected into a separate class (e.g. `ParticipantSummaryService` or `DeceasedReportsService`) rather than called directly from a controller.